### DISCUSSION TITLE
Doc Verifications

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#4545] [Ideas]
It would be helpful to have the option to verify docs and set the frequency of review. This helps ensure that specific pages are up to date. This feature acts as an audit for docs that have dynamic content. Slab has this feature, and I have found it very useful. I included a few screenshots for reference. 

![CleanShot 2022-12-05 at 22 40 39](https://user-images.githubusercontent.com/96745845/205807864-9649b68a-f0f8-40fe-ae16-b366594e4255.png)
![CleanShot 2022-12-05 at 22 38 25](https://user-images.githubusercontent.com/96745845/205807882-246c064b-f3fa-4ecc-a720-1dcdb91a5a60.png)
![CleanShot 2022-12-05 at 22 38 07](https://user-images.githubusercontent.com/96745845/205807887-cca2e8ba-9f99-4beb-8e54-785e2afffc56.png)
![CleanShot 2022-12-05 at 22 38 00](https://user-images.githubusercontent.com/96745845/205807893-ff274c46-d4a6-496d-89d6-ee7baf609b88.png)

**Comment by tommoor** [comment]
Totally, this definitely feels within scope of the project.

**Reply by edwarddarrah** [reply]
@tommoor That is great to hear!

**Reply by edwarddarrah** [reply]
@tommoor I am checking in to see if there has been any movement with this feature. Thanks.

**Reply by tommoor** [reply]
No movement yet, last few months have been building comments, mentions, and notifications.

**Reply by edwarddarrah** [reply]
@tommoor Thanks for the reply. The updates you mentioned have been helpful in improving my team's collaboration.

**Reply by Shuttleu** [reply]
@tommoor Don't suppose you've had a chance to think about adding this yet?

**Reply by tommoor** [reply]
No progress, unfortunately other features always take priority and the team is tiny

**Comment by zesaver** [comment]
+1 here

**Comment by jamespanic** [comment]
We're building up quite a bit of documentation in outline and it would be helpful to have the ability to schedule reviews of documents. 

My full wishlist would be:

- At an instance level I might want every doc to be reviewed at least every six months with some docs requiring monthly review.
- There would need to be someone assigned as the reviewer. 
- The system should send out an email reminder that a page needs reviewing
- Pages needing review should be badged so readers know the info may be out of date.
- Send out weekly reminders to review pages
- Provide a review queue so users can easily go through pages assigned to them
